src/test/java/it/pagopa/transactions/client/NodeForPspClientTest.java

```diff
@@ -220,6 +230,51 @@ void shouldReturnOKClosePaymentResponse() {
         assertThat(clientResponse.getOutcome()).isEqualTo(closePaymentResponse.getOutcome());
     }
 
+    @Captor
+    ArgumentCaptor<Publisher<ClosePaymentRequestV2Dto>> bodyCaptor;
+
+    @Test
+    void shouldReturnOKClosePaymentResponseAdditionalInfo() {
```

Comment:
I would rather add two tests here on the serialized ClosePaymentRequest string.
One test where additionalPaymentInformations RRN is valued and another one were is not and check that RRN is serialized in the first and not in the second one.

Maybe for let this being testable the used ObjectMapper used to build Webclient can be moved into a producer method inside WebClientsConfig class

Reply:
Thanks, done with 88c9bb9 👍